### DISCUSSION TITLE
Add "Seasons" count to TV Overview

### DISCUSSION THREAD

**Discussion by cosmosxcii** [sct/overseerr#1075] [Feature Requests]
Currently, when in a TV show's overview page, the user has to press "Request" to see how many seasons the show has. This could be useful information to place in the right hand column.

I've mocked how this could look
![SeasonsCount](https://user-images.githubusercontent.com/76740954/109852407-a7d45000-7c4c-11eb-88ea-ad16a8a3683c.jpg)

**Comment by TheCatLady** [comment]
@cosmosxcii I have an open PR for adding the season count: https://github.com/sct/overseerr/pull/1078

Also, GitHub Discussions is not the best place for feature/enhancement requests.  Please submit those requests to our [issue tracker](https://github.com/sct/overseerr/issues/new/choose) so that they can be triaged and handled appropriately.
